pkg/builders/catalogsourceconfigbuilder.go

```diff
@@ -1,40 +1,26 @@
-package operatorsource
+package builders
 
 import (
 	"fmt"
 
 	marketplace "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
+	"github.com/operator-framework/operator-marketplace/pkg/datastore"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// DatastoreLabel is the label used in a CatalogSourceConfig to indicate that
-// the resulting CatalogSource acts as the datastore for the OperatorSource
-// if it is set to "true".
-const DatastoreLabel string = "opsrc-datastore"
-
-// OpsrcOwnerNameLabel is the label used to mark ownership over resources
-// that are owned by the OperatorSource. When this label is set, the reconciler
-// should handle these resources when the OperatorSource is deleted.
-const OpsrcOwnerNameLabel string = "opsrc-owner-name"
-
-// OpsrcOwnerNamespaceLabel is the label used to mark ownership over resources
-// that are owned by the OperatorSource. When this label is set, the reconciler
-// should handle these resources when the OperatorSource is deleted.
-const OpsrcOwnerNamespaceLabel string = "opsrc-owner-namespace"
-
 // CatalogSourceConfigBuilder builds a new CatalogSourceConfig type object.
 type CatalogSourceConfigBuilder struct {
-	object marketplace.CatalogSourceConfig
+	Object marketplace.CatalogSourceConfig
```

Comment:
What is the reason for this change?

Reply:
Otherwise we cannot set the object here https://github.com/operator-framework/operator-marketplace/pull/198/files/d59caf31fca43b6ee33649a8da90d55e69a4f615..c8dabbe50a7b43dcef2e54d7bf3792fbe1daeb61#diff-20a79a6a9296f6b57de79782f072fbb6R216. Previously since the catalogsourceconfigbuilder was in the same package as the operatorsource this wasn't an issue.

Reply:
Please mention that in the commit message